src/CssEmbed/CssEmbed.php

```diff
@@ -73,6 +73,12 @@ protected function replace($matches)
      */
     protected function embedFile($file)
     {
+		// Check for various weird formats which the system is unable to handle. Perhaps there is a better way at some point?
+	    if (substr($file, -4) == '.eot' || substr($file, -4) == '.ttf' || substr($file, -5) == '.woff')
```

Comment:
I'll build a test, making sure it's not processing anything with a weird (so far, font-type), file ending and fix indentation